src/components/dropdown-select.js

```diff
@@ -63,7 +63,11 @@ const DropdownSelect = ({
   const [selectLimit, setSelectLimit] = useState(limit || 0)
   const [open, setOpen] = useState(false)
   const { ref, componentIsActive, setComponentIsActive } = useComponentIsActive()
-  
+
+  useEffect(() => {
+    setSelectedOptions(setSelectedOption || [])
```

Comment:
can we re-name the `setSelectedOption` prop to `selectedOptions` or `values`?

maybe also differentiate between controlled vs uncontrolled to address this issue.
e.g. when uncontrolled the selected-values would be set from within the component, and `setSelectedOption` prop would be used as a `defaultValues` prop (suggest to add this prop in as well), and when controlled the value is set externally of the component instead of internally (ie controlled by user)

Reply:
I think this is a great idea. @kc-leung what do you think?

Reply:
@hyx131 @kc-leung I've pushed some changes to match Selin's idea.

Reply:
thanks, looks good